Add "gh repo create" to SETUP.md (#419)
manubot/rootstock: SETUP.md

git remote set-url origin git@github.com:$OWNER/$REPO.git
```

> Then you need to create an empty GitHub repository. 
> You can do that at <https://github.com/new> or via the  [GitHub command line interface](https://github.com/cli/cli):

Reviewer: ```suggestion
Then create an empty repository on GitHub. 
You can do this at <https://github.com/new> or via the [GitHub command line interface](https://github.com/cli/cli) (if installed) with `gh repo create`.
```

If accepting this suggestion, delete the following code block

```sh

> $ gh repo create

Reviewer: we tend not to prefix terminal commands with `$`, since it makes it harder to copy paste the entire code block

> ```sh
> $ gh repo create
> ```

Reviewer 2: ```suggestion
```
This code block can be removed now that the command is inline.
